Formula/argo.rb

```diff
@@ -0,0 +1,30 @@
+class Argo < Formula
+  desc "Get stuff done with container-native workflows for Kubernetes"
+  homepage "https://argoproj.io"
+  url "https://github.com/argoproj/argo.git",
+      :tag      => "v2.9.3",
+      :revision => "9407e19b3a1c61ad4043e382484fd0b6b15574f2"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "yarn" => :build
+
+  def install
+    inreplace "Makefile", "CGO_ENABLED=0", ""
+    system "make", "dist/argo-darwin-amd64"
```

Comment:
Could we somehow make this step more OS-agnostic? I mean, this will probably fail on Linux.

Reply:
@dawidd6 I must have overlooked the more general target `dist/argo` which I now use.